pkg/collect/host_sysctl.go

```diff
@@ -0,0 +1,45 @@
+package collect
+
+import (
+	"bytes"
+	"os/exec"
+
+	"github.com/pkg/errors"
+	troubleshootv1beta2 "github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta2"
+)
+
+var _ HostCollector = (*CollectHostSysctl)(nil)
```

Comment:
```suggestion
// Ensure `CollectHostSysctl` implements `HostCollector` interface at compile time.
var _ HostCollector = (*CollectHostSysctl)(nil)
```

+1 on doing this. I added a comment for future travellers